src/Algebra/Construct/DirectProduct.agda

```diff
@@ -151,3 +161,89 @@ abelianGroup G H = record
     ; comm = λ x y → (G.comm , H.comm) <*> x <*> y
     }
   } where module G = AbelianGroup G; module H = AbelianGroup H
+
+isSemiringWithoutAnnihilatingZero :
+  (R : SemiringWithoutAnnihilatingZero a ℓ₁) →
+  (S : SemiringWithoutAnnihilatingZero b ℓ₂) →
+  let module R = SemiringWithoutAnnihilatingZero R
+      module S = SemiringWithoutAnnihilatingZero S
+  in
+  IsSemiringWithoutAnnihilatingZero (Pointwise R._≈_ S._≈_) (zip R._+_ S._+_)
```

Comment:
@mechvel thanks for the PR. However take a look at how the other proofs in this file are structured. Your new proofs do not fit the same pattern. Please fix.

Reply:
May be now it does fit?
I have also fixed CHANGELOG.md. 
But `src/Algebra/Morphism/Structures.agda` is still detected as changed. It needs to be of the current `master`, 
currently I do not need to touch it.

Reply:
> May be now it does fit?

Yes much better.

> But src/Algebra/Morphism/Structures.agda is still detected as changed. It needs to be of the current master,
currently I do not need to touch it.

This has been fixed on `master` already. If you merge `master` into this branch then it'll disappear. Alternatively you can wait until we merge this into `master` and it will also disappear.